src/main/java/at/hannibal2/skyhanni/features/fishing/trophy/TrophyFishManager.kt

```diff
@@ -23,6 +26,46 @@ object TrophyFishManager {
     val fishes: MutableMap<String, MutableMap<TrophyRarity, Int>>?
         get() = ProfileStorageData.profileSpecific?.crimsonIsle?.trophyFishes
 
+    private var loadedNeu = false
+
+    @SubscribeEvent
+    fun onNeuPVDataLoaded(event: ProfileDataLoadedEvent) {
+        if (!LorenzUtils.inSkyBlock) return
+
```

Comment:
idk, but I feel like there is an isEnabled of some kind missing.

Reply:
maybe instead of an isEnabled send a clickable chat message prompting the data to be updated?

Reply:
Implemented at https://github.com/hannibal002/SkyHanni/pull/1128

Reply:
@Thunderblade73 @martimavocado